src/handlers/query-wallet.ts

```diff
@@ -0,0 +1,86 @@
+import { ethers } from "ethers";
+import { Context } from "../types";
+
+function extractEnsName(text: string) {
+  const ensRegex = /^(?=.{3,40}$)([a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z]{2,}$/gm;
+  const match = text.match(ensRegex);
+
+  if (match) {
+    const ensName = match[0];
+    return ensName?.toLowerCase();
+  }
+}
+
+export async function registerWallet(context: Context, body: string) {
+  const payload = context.payload;
+  const config = context.config;
+  const logger = context.logger;
+  const sender = payload.sender.login;
+  const adapters = context.adapters;
+
+  const regexForAddress = /(0x[a-fA-F0-9]{40})/g;
+  const addressMatches = body.match(regexForAddress);
+  let address = addressMatches ? addressMatches[0] : null;
+  const ensName = extractEnsName(body.replace("/wallet", "").trim());
+
+  if (!address && ensName) {
+    context.logger.debug("Trying to resolve address from ENS name", { ensName });
+    address = await resolveAddress(ensName);
+    if (!address) {
+      throw new Error(`Resolving address from ENS name failed: ${ensName}`);
+    }
+    context.logger.debug("Resolved address from ENS name", { ensName, address });
+  }
+
+  if (!address) {
+    return context.logger.info("# Skipping to register a wallet address because both address/ens doesn't exist");
```

Comment:
```suggestion
    return context.logger.info("Skipping to register a wallet address because both address/ens doesn't exist");
```
shouldnt need to prepend hashtag